components/blocks/expertBlock.tsx

```diff
@@ -0,0 +1,115 @@
+import Image from "next/image";
+import { type Template } from "tinacms";
+import { CustomLink } from "../customLink";
+
+export const ExpertBlock = (props) => {
+  const { header, expertList, link } = props;
+  return (
+    <>
+      <hr />
+      <h2 className="pb-12">{header.toUpperCase()}</h2>
```

Comment:
```suggestion
      <h2 className="pb-12">{header && header.toUpperCase()}</h2>
```
guarding against missing data